src/sentry/utils/otp.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import absolute_import
+
+import time
+import hmac
+import random
+import base64
+import qrcode
+import urllib
+import hashlib
+from datetime import datetime
+from itertools import izip
+
+from sentry.utils.dates import to_timestamp
+
+
+_builtin_constant_time_compare = getattr(hmac, 'compare_digest', None)
+
+
+def constant_time_compare(val1, val2):
```

Comment:
Django has this already. `from django.utils.crypto import constant_time_compare`. Let's just use that.